components/automate-cli/pkg/verifyserver/server/api/v1/batchcheck_test.go

```diff
@@ -1,6 +1,7 @@
 package v1_test
 
 import (
+	"errors"
 	"io/ioutil"
```

Comment:
`ioutil.ReadAll(res.Body)` is deprecated use  `io.ReadAll(res.Body)` instead on line 193

Reply:
Done